src/index.js

```diff
@@ -358,19 +358,18 @@ function createConfig(options, entry, format, writeMeta) {
 							},
 						}),
 					!useTypescript && flow({ all: true, pretty: true }),
-					nodent({
-						exclude: 'node_modules/**',
-						noRuntime: true,
-						promises: true,
-						transformations: {
-							forOf: false,
-						},
-						parser: {
-							plugins: {
-								jsx: true,
-							},
-						},
-					}),
+					// Only used for async await
+					!useTypescript &&
```

Comment:
Should TypeScript use the babel transform or it's own generator-styled one? Should be able to have TypeScript target ESNext or ES2017, and then feed into babel for it to do JSX and async/await transforms

Reply:
Ohh that's a great idea 👍💯 I'll try it out

Reply:
Added it 🎉 Setting the target to `es2017` is necessary because TS uses some things that are not supported yet in preset-env as far as I'm aware.